opencl/source/cl_device/cl_device.cpp

```diff
@@ -205,4 +205,9 @@ bool ClDevice::arePipesSupported() const {
     return device.getHardwareInfo().capabilityTable.supportsPipes;
 }
 
+bool ClDevice::isPciBusInfoValid() const {
+    return deviceInfo.pciBusInfo.pci_domain != PhysicalDevicePciBusInfo::InvalidValue && deviceInfo.pciBusInfo.pci_bus != PhysicalDevicePciBusInfo::InvalidValue &&
+           deviceInfo.pciBusInfo.pci_device != PhysicalDevicePciBusInfo::InvalidValue && deviceInfo.pciBusInfo.pci_function != PhysicalDevicePciBusInfo::InvalidValue;
+}
+
```

Comment:
Do we have ULT that can verify each condition will be catched if PCI Bus is invalid (and when is valid also)

Reply:
The tests have been added. Please see new tests in opencl/test/unit_test/device/device_tests.cpp